src/main/java/org/alliancegenome/curation_api/jobs/util/SlackNotifier.java

```diff
@@ -22,7 +22,7 @@
 @ApplicationScoped
 public class SlackNotifier {
 	
-	@ConfigProperty(name = "net")
+	@ConfigProperty(name = "net", defaultValue="\"\"")
```

Comment:
I am guessing this is a change that came in from another PR? I think this should be `, defaultValue=""`

Reply:
Or just add `NET = dev` in your application.properties file.

Reply:
it was put in to avoid endlessly looping errors when developing locally - it's to avoid entering the block of code that starts with `if(!systemName.get().equals("\"\"") && !slackToken.get().equals("\"\"")) {`
Does adding `NET=dev` do that?

Reply:
Well the NET=dev sets it to a value as I would guess maybe your file is missing it?